cmd/cosign/cli/generate/generate.go

```diff
@@ -49,6 +48,6 @@ func GenerateCmd(ctx context.Context, regOpts options.RegistryOptions, imageRef
 	if err != nil {
 		return err
 	}
-	fmt.Fprint(w, string(json))
+	w.Write(json)
```

Comment:
Is this equivalent to `w.Write(string(json))`? I assume that if this were printed to stdout, it needs to be a string?

Reply:
I thought it doesn't matter whether you write out bytes from a byte array, or the string "wrapped around" those bytes - the string conversion is an unnecessary performance degradation (if not eliminated by the compiler).
Writing bytes to stdout seems to be fine (without the `string` conversion): https://go.dev/play/p/tPrMBjXyxvq.